### ISSUE TITLE
Could you add a training file?

### ISSUE THREAD

**Issue by wulikai1993** [StanislasBertrand/RetinaFace-tf2#6]
And how long time it takes to train a model?
Do you train the model on a single GPU or multi GPUs?

**Comment by StanislasBertrand** [comment]
Hi,

There is no training method provided. I converted the already trained weights and code from 
https://github.com/deepinsight/insightface/tree/master/RetinaFace

Although the provided model is a classif tf.keras model, you could try to train it the usual tf.keras way. In order to match the articles method you would probably also need to implement / find a tf.keras implementation of their multi task loss
